content/guides/models/track/project-page.md

```diff
@@ -183,6 +180,10 @@ By default, this turns other numeric columns into histograms showing the distrib
 
 <!-- ## Automations tab -->
 
+## Automations tab
```

Comment:
It would be good to add a link to the new Automations landing page when https://github.com/wandb/docs/pull/1084 lands.